apps/blestress/src/tx_stress.c

```diff
@@ -848,6 +848,16 @@ tx_stress_9_gap_event(struct ble_gap_event *event, void *arg)
 {
     ble_addr_t addr;
     int test;
+    struct ble_gap_conn_params conn_params = {
+        .scan_itvl = 0x0010,
+        .scan_window = 0x0010,
+        .itvl_min = BLE_GAP_INITIAL_CONN_ITVL_MIN,
+        .itvl_max = BLE_GAP_INITIAL_CONN_ITVL_MAX,
+        .latency = 0,
+        .supervision_timeout = 0x0C80,
+        .min_ce_len = 0x0010,
+        .max_ce_len = 0x0300,
+    };
```

Comment:
are those connections maintained in test for at least this timeout after last one is created?

Reply:
@KKopyscinski ?

Reply:
There are total of 15 successful concurrent connections. For example run, last succesfull connection occurs ~164343728us timestamp and first disconnect at 196078120us. This gives us 31.734392s. 0x0C80 is 3200, x10ms = 32s, which considering print delays seems pretty close IMO.